global_planner/src/orientation_filter.cpp

```diff
@@ -35,19 +35,17 @@
  * Author: David V. Lu!!
  *********************************************************************/
 #include <global_planner/orientation_filter.h>
-#include <tf/tf.h>
+#include <tf2/utils.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 #include <angles/angles.h>
 
 namespace global_planner {
 
 void set_angle(geometry_msgs::PoseStamped* pose, double angle)
 {
-    pose->pose.orientation = tf::createQuaternionMsgFromYaw(angle); 
-}
-
-double getYaw(geometry_msgs::PoseStamped pose)
-{
-    return tf::getYaw(pose.pose.orientation);
+  tf2::Quaternion q;
+  q.setEuler( angle, 0, 0 );
```

Comment:
again, I think this should be SetRPY possibly?

Reply:
I can verify that changing `q.setEuler( angle, 0, 0 );` to `q.setRPY(0, 0, angle);` works